nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/db/event/ColumnDefinition.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.processors.standard.db.event;
+
+/**
+ * A class that specifies a definition for a relational table column, including type, name, etc.
+ */
+public class ColumnDefinition {
+
+    private byte type;
+    private String name = "";
+
+    public ColumnDefinition(byte type) {
```

Comment:
The type values may change depending on the implementation (MySQL, Oracle, e.g.) although hopefully they are all using [JDBC Types](http://docs.oracle.com/javase/8/docs/api/constant-values.html#java.sql.Types.BIT). With that said, I can't for the life of me remember why I chose "byte" instead of int; the only place it's stored (currently) is from a ResultSetMetaData object which returns an int for getColumnType().

I will change the type to an int and then folks can use the JDBC Type constants in code if they like. How does that sound?